konvoy: document race condition for pre-provisioned adoption (#4070)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md

The cluster, control plane, and worker node pool should all show the value `True` in the Ready column.

### If the Cluster Never Reaches a Steady State

> It is possible that the cluster never reaches a steady state. This is caused by a race condition where the `Machine` objects are reconciled before the appropriate status is set on the `Cluster` object and it gets "stuck".

Reviewer: ```suggestion
A race condition where the `Machine` objects are reconciled before the appropriate status is set on the `Cluster` object and it gets "stuck", stopping it from reaching a steady state. If this happens, delete the CAPPP pod and let Kubernetes restart it to trigger a reconcile:
```

> To trigger a reconcile, delete the CAPPP pod and let Kubernetes restart it:

Reviewer: ```suggestion

```

> 
> To trigger a reconcile, delete the CAPPP pod and let Kubernetes restart it:

Author: ```suggestion
```

```sh
kubectl --kubeconfig=admin.conf delete pods -n cappp-system -l control-plane=controller-manager
```

> The output appears similar to this:

Reviewer: ```suggestion
The following output appears:
```

```sh
pod "cappp-controller-manager-56fcf85446-66z87" deleted
```

> ## Wait for Calico to be upgraded

Reviewer: ```suggestion
## Wait for Calico to upgrade
```
